src/test/denialofservice_tests.cpp

```diff
@@ -190,6 +190,70 @@ BOOST_AUTO_TEST_CASE(stale_tip_peer_management)
     connman->ClearTestNodes();
 }
 
+BOOST_AUTO_TEST_CASE(block_relay_only_eviction)
+{
+    const CChainParams& chainparams = Params();
+    auto connman = std::make_unique<ConnmanTestMsg>(0x1337, 0x1337, *m_node.addrman);
+    auto peerLogic = PeerManager::make(chainparams, *connman, *m_node.addrman, nullptr,
+                                       *m_node.chainman, *m_node.mempool, false);
+
+    constexpr int max_outbound_blocksonly = MAX_BLOCK_RELAY_ONLY_CONNECTIONS;
```

Comment:
```suggestion
    constexpr int max_outbound_blockrelay{MAX_BLOCK_RELAY_ONLY_CONNECTIONS};
```

Might be better to call this blockrelay to avoid confusion with `-blocksonly`.

Reply:
changed to `max_outbound_block_relay`